docs/modules/plugins/partials/locator-header.adoc

```diff
@@ -13,3 +13,42 @@ By. prefix is optional.
 . `visibility` - *[optional]* visibility of element (visible by default)
 . `filterType` - *[optional]* type of the filter
 . `filterValue` - *[required if filter type defined]* value of the filter
+
+
+=== Reusable locators
+
+Feature allows to define custom reusable locator types via properties. These new locator types should rely on one of the already existing ones.
+
+.Properties
+----
+ui.locator.<locatorName>.pattern
+ui.locator.<locatorName>.locator-type
+----
+
+. `locatorName` - The name of the reusable locator.
+. `pattern` - The pattern of the reusable locator. Parameter placeholder is `%s`
```

Comment:
also it makes sense to put info on how to escape `%` in case if it shouldn't be used as a part of the placeholder